plugin/tataku.vim

```diff
@@ -8,11 +8,14 @@ set cpo&vim
 
 if get(g:, 'tataku_enable_operator', v:false)
   for recipe in keys(get(g:, 'tataku_recipes', {}))
-    call operator#user#define(
-          \ 'tataku-' .. recipe,
-          \ 'tataku#_call_as_operator',
-          \ printf('call tataku#_setup_operator("%s")', recipe)
-          \ )
+    for mode in ['n', 'v']
```

Comment:
hmm...
I think xnoremap is better than vnoremap.
Beause vnoremap is include visual mode and select mode too.
ref: `:h map-table`

Reply:
Fix at 0cf5870